src/core/enums/ActionType.enum.h

```diff
@@ -29,6 +29,7 @@ enum ActionType {
     // Menu file
     ACTION_NEW = 100,
     ACTION_OPEN,
+    ACTION_CHANGE_BACKGROUND,
```

Comment:
I suggest to rename this variable to `ACTION_CHANGE_PDF_BACKGROUND`.

Reply:
done